step.yml

```diff
@@ -52,6 +52,20 @@ inputs:
       description: |-
         The commit hash for the commit we are working with
       is_required: true
+  - target_url: "$BITRISE_BUILD_URL"
+    opts:
+      title: "Target URL"
+      summary: ""
+      description: |-
+        The target URL to associate with this status.
+        This URL will be linked from the GitLab UI to allow users to easily see the source of the status.
+  - context: "Bitrise"
+    opts:
+      title: "Context"
+      summary: "The Identifier to be used for this status"
+      description: |-
+        A string label to differentiate this status from the status of other systems.
```

Comment:
only double newline will be displayed as truly new line in the step input's fields. If you need newline, please use two of them. (MD)